codacy/values-production.yaml

```diff
@@ -295,6 +295,15 @@ engine:
     requests:
       cpu: 200m
       memory: 500Mi
+codacy-tools:
+  replicaCount: 1
+  resources:
```

Comment:
It should be fine to request more resources here, because we can assume a larger cluster is available, so these limits should be ok.